core/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -425,4 +447,28 @@ public boolean isClosed() {
     public Injector injector() {
         return this.injector;
     }
+
+    @SuppressForbidden(reason = "Need to specify port and pass to formatter")
```

Comment:
Its kind of difficult to tell during the review which forbidden API you are calling. Maybe make a small method that has the annotation so its super obvious and nothing sneaks in you don't expect?